src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -54,7 +54,7 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
 
             // Order the documents by FilePath.  Default ordering in the RemoteWorkspace is
```

Comment:
for C# and VB, I am not sure how ordering affects correctness. for other language, might be. but right now, it only support C# and VB.

I chose the first since I didn't need new API from Workspace to do so. otherwise, probably need new API since current Add only append new item to the end of list. I need Insert to put item at certain location of existing projects/documents list.